src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/collection-create-overlay/main.js

```diff
@@ -95,28 +95,32 @@ define(['services/sulumedia/collection-manager'], function(CollectionManager) {
         },
 
         /**
-         * Creates a new Collections with the form-data of the overlay
-         * @returns {Boolean} returns false if form-data was not valid
+         * Validates the overlay form-data and saves collection if form-data is valid
+         * @returns {boolean} false if form-data is not valid
          */
-        addCollection: function() {
+        okCallback: function() {
             if (this.sandbox.form.validate(constants.newFormSelector)) {
-                var collection = this.sandbox.form.getData(constants.newFormSelector);
-                collection.parent = this.options.parent;
-
-                CollectionManager.save(collection).then(function(collection) {
-                    this.sandbox.emit('sulu.media.collection-create.created', collection)
-                    this.sandbox.stop();
-                }.bind(this)).fail(function() {
-                        this.sandbox.stop();
-                    }.bind(this)
-                );
-                return true;
+                this.addCollection();
+                this.sandbox.stop();
             } else {
                 return false;
             }
         },
 
         /**
+         * Creates a new Collections with the form-data of the overlay
+         * @returns {Boolean} returns false if form-data was not valid
+         */
+        addCollection: function() {
+            var collection = this.sandbox.form.getData(constants.newFormSelector);
+            collection.parent = this.options.parent;
+
+            CollectionManager.save(collection).done(function(collection) {
+                Mediator.emit('sulu.media.collection-create.created', collection);
```

Comment:
@wachterjohannes the sandbox is already closed when this is executed (see line 104)